src/filter.spec.js

```diff
@@ -15,6 +15,7 @@ import filterFixture10 from "./__tests__/filterFixture10.json";
 import filterFixture11 from "./__tests__/filterFixture11.json";
 import filterFixture12 from "./__tests__/filterFixture12.json";
 import filterFixture13 from "./__tests__/filterFixture13.json";
+import filterFixtureA3AndA4 from "./__tests__/filterFixtureA3AndA4.json";
```

Comment:
Folder structure and having tests next to source made it difficult for me to overlook the project and to see the project structure at a glance. Don't like it. But that's just a comment.